lib/game_runner.rb

```diff
@@ -0,0 +1,20 @@
+require_relative "./board.rb"
```

Comment:
When using `require_relative`, you can omit the `./`.

Reply:
> Do you see the board if you run the app with `ruby lib/game_runner.rb`?

hmm I was at some point but am not any more. Let me look into it.